wasp/apps/flashlight.py

```diff
@@ -21,19 +20,31 @@ class TorchApp(object):
 
     def foreground(self):
         """Activate the application."""
-        self.draw()
         wasp.system.request_tick(1000)
+        wasp.system.request_event(wasp.EventMask.TOUCH)
 
         self._brightness = wasp.system.brightness
         wasp.system.brightness = 3
+        self.n_touch = 0
+        self.draw()
 
     def background(self):
-        """De-activate the application (without losing state)."""
+        """De-activate the application (without losing original state)."""
         wasp.system.brightness = self._brightness
 
     def tick(self, ticks):
         wasp.system.keep_awake()
 
     def draw(self):
         """Redraw the display from scratch."""
-        wasp.watch.drawable.fill(0xffff)
+        if self.n_touch % 6 < 3:
+            wasp.watch.drawable.fill(0xffff)
+        else:
+            wasp.watch.drawable.fill(0xf800)
+        print("ok")
+
+    def touch(self, event):
+        self.n_touch += 1
+        n = (wasp.system.brightness - 1) % 3
+        wasp.system.brightness = n if n else 3
```

Comment:
why the if? aren't you already changing the brightness by 1?

Reply:
The issue is that when it goes to 0 the watch screen turns off and the watch goes back to home. I couldn't find a clever way to code it :/

Reply:
This should do it:

```python
wasp.system.brightness = (wasp.system.brightness % 3) + 1
```

Reply:
oops, it goes the wrong way. you want it to decrease, right?

Reply:
I slightly modified mine in 12df35b, thanks